src/components/WalletStatementModal/index.js

```diff
@@ -41,9 +41,7 @@ class WalletStatementModal extends React.Component {
         const authToken = lodashGet(this.props, 'session.authToken', null);
         return (
             <>
-                <FullScreenLoadingIndicator
-                    visible={this.state.isLoading}
-                />
+                {this.state.isLoading && <FullScreenLoadingIndicator />}
```

Comment:
**thought (not blocking):** Looks like the video where we saw the long spinner was after a login so it's probably not this one.

Reply:
There are some others here where we can probably assume the same. But maybe we should just always log for all slow spinners with or without additional context.

Reply:
Ah ok I see now. And it is easier to remove the prop and track mount and unmount vs. changes to `props.isLoading`

Reply:
Nice change 👍